src/main/java/net/skinsrestorer/bukkit/skinfactory/SkinFactory.java

```diff
@@ -35,8 +36,16 @@
      * @param props - Property Object
      */
     default void applySkin(final Player p, Object props) {
```

Comment:
Need to be tested

Reply:
It still applies for me on bungeecord, so i think its fine.